source/localizable/smart_cart/_order_object.html.md.erb

```diff
@@ -13,13 +13,26 @@ Name         | Type   | Value | Description
 `dispatch_until` | Date String in format<br> `YYYY-MM-DDTHH:MM:SS+HH:M` | | Order maximum dispatch date
 `accept_options` | Object | | [Accept options](#accept-options) (available for orders with state `"open"`)
 `reject_options` | Object | | [Reject options](#reject-options) (available for orders with state `"open"`)
+`invoice`        | Boolean | | Indicates invoice request for the order
+`invoice_details` | Object | | [Invoice details](#invoice-details) - if invoice requested
 
 #### Customer details
 
 Name                              | Type   | Description
 :-------------------------------: | :----: | :------------------------------------------:
 `customer.first_name`             | String |First name
 `customer.last_name`              | String |Last name
+`customer.address`                | Object |Order shipping address
+
+### Customer address object
```

Comment:
Let's not use H3 here. another H4 would be fine

![image](https://user-images.githubusercontent.com/1265295/84034927-ac145b00-a9a3-11ea-8454-0592ed307d6c.png)

Also, we could name it "Customer address"

Reply:
This was by mistake :) +1 for the description!